public/env.json

```diff
@@ -1,5 +1,6 @@
 {
     "BASE_API_URL": "",
+    "FAUCET_URL": "https://faucet.dev.sora2.tachi.soramitsu.co.jp/",
```

Comment:
You should also add PR to `sora2-deploy` repo with the following updates for each stand

Reply:
@stefashkaa check please is my PR correct? https://github.com/soramitsu/sora2-deploy/pull/36

Reply:
Cool